content/en/docs/setup/independent/high-availability.md

```diff
@@ -339,7 +339,7 @@ Run the following to generate the manifest file:
       EOF
 
 Make sure you replace:
-* `<podname>` with the name of the node you're running on (e.g. `etcd0`, `etcd1` or `etcd2`)
+* `<podname>` and `<name>` with the name of the node you're running on (e.g. `etcd0`, `etcd1` or `etcd2`)
```

Comment:
probably best to edit the example above to use the same placeholder in that case.
either `<podname` or `<name>`...